.circleci/config.yml

```diff
@@ -126,6 +126,7 @@ jobs:
         - python/install-packages:
             pkg-manager: "pip"
             pip-dependency-file: "requirements-gsutil.txt"
+        - <<: *install_node_ci
```

Comment:
Why is this here? The `install_demisto_sdk` executed on line 125 includes this reference configuration.